aexpect/remote_door.py

```diff
@@ -696,12 +735,31 @@ class ModuleObject:  # pylint: disable=R0903
         """Module wrapped for transferability."""
 
     for fname, fobj in inspect.getmembers(module, inspect.isfunction):
-        if not whitelist or (object_name, fname) in whitelist:
+        if not object_wl[0] or fname in object_wl[0]:
```

Comment:
IIUC the `whitelist` used to be `[(module1, function1), (module2, function2)]` and here you're changing it to `[(module1, module2), (function1, function2)]`?

Reply:
Rather to `([function1, function2], [module1, module2])` as documented on line 658.

Reply:
As the user typically access a remote object and this internally calls the sharing of a local object, there are few calls that have previously depended on this argument and the current change is putting it in actual use.

Reply:
No worries! Resolving this